sdk/batch/Microsoft.Azure.Management.Batch/src/Microsoft.Azure.Management.Batch.csproj

```diff
@@ -8,7 +8,7 @@
     <Description>Provides management capabilities for Azure Batch service accounts.</Description>
     <AssemblyTitle>Microsoft Azure Batch Management Library</AssemblyTitle>
     <AssemblyName>Microsoft.Azure.Management.Batch</AssemblyName>
-    <Version>13.0.0</Version>
+    <Version>14.0.0</Version>
```

Comment:
You don't need to update the major version, this could be 13.1.0, but this is up to you.  regardless, this version should match the assembly and assemblyFileVersion marked below.